README.md

```diff
@@ -10,20 +10,20 @@ Make sure that:
 * You have `kinit`-ed.
 * You have added the public SSH key, often `~/.ssh/id_rsa.pub`, from your machine to [GitHub](https://github.com/settings/keys)
 * You have previously `ssh`-ed into all servers.
-  * NOTE: If you are unsure about this, run `./deploy.rb [qa|stage|prod] --checkssh` and watch the output for any errors!
+  * NOTE: If you are unsure about this, run `thor check_ssh -e [qa|stage|prod]` and watch the output for any errors!
 
 ### Check your ssh connection to all servers
 
 ```
-./deploy.rb [qa|stage|prod] --checkssh
+thor check_ssh -e [qa|stage|prod]
```

Comment:
How about we keep `deploy` and do like sinopia_cli does, and invoke Thor: https://github.com/LD4P/sinopia_cli/blob/main/exe/sinopia-cli & https://github.com/LD4P/sinopia_cli/blob/main/lib/sinopia_cli.rb#L8

Reply:
@jcoyne are you suggesting we have one `exe` per operation (`deploy`, `check_cocina`, `check_ssh`) or that we route all three through `deploy`?

Reply:
@jcoyne I ask bc `deploy check_cocina` realllllly doesn't do it for me. (Checking cocina is a local operation.)

Reply:
@jcoyne Made a related change. Have another look?